### ISSUE TITLE
Please help me poor quality on custom dataset, wandb looks fine

### ISSUE THREAD

**Issue by hanjoonwon** [autonomousvision/sdfstudio#277]
1. process data with nerfstudio 
ns-process-data images --data /battery --output-dir /workspace/battery 

2. python3 /sdfstudio/scripts/datasets/process_nerfstudio_to_sdfstudio.py  --mono-prior --data-type colmap --scene-type object --data /battery --output-dir /workspace/batterydata  --omnidata-path /omnidata/omnidata_tools/torch --pretrained-models /omnidata/omnidata_tools/torch/pretrained_models 
get normal,rgb,depth

3.i trained with ns-train neus-facto --pipeline.datamanager.train-num-rays-per-batch 1024 --pipeline.model.mono-depth-loss-mult 0.0 --pipeline.model.mono-normal-loss-mult 0.0 --vis wandb --pipeline.model.sdf-field.inside-outside False --pipeline.model.background-model mlp --trainer.steps_per_save 5000  --trainer.max-num-iterations 30000 --experiment-name facbattery sdfstudio-data --data /workspace/batterydata

My environment was run using docker on wsl2 ubuntu22.04 on windows10 Graphics card is RTX2060
it's my meta
[meta_data.json](https://github.com/autonomousvision/sdfstudio/files/13927761/meta_data.json)
data

![image](https://github.com/autonomousvision/sdfstudio/assets/125524043/ce2224a6-69ee-49d6-91d7-aa6f49217485)

![image](https://github.com/autonomousvision/sdfstudio/assets/125524043/6f795f30-df8f-452a-936a-141273b81b80)

https://wandb.ai/ju805604/sdfstudio/runs/r7wtqije?workspace=user-ju805604

**Comment by davidceka** [comment]
You're running the model on a rtx 2060 and it ran for only 20k iterations. I don't know about neus-facto but training with bakedangelo needs at least 200k~ iterations to have some good results. Have you tried training for longer?

**Comment by davidceka** [comment]
Also, you can tell that the model didn't quite learn the shape of the object since the evaluations are really fuzzy (left is the original, right is the predicted one) and that is also confirmed by the depth image directly on top, all is purple so i think the model needs more training
![image](https://github.com/autonomousvision/sdfstudio/assets/90139776/688ba8d2-7792-4903-8fa8-05c0cffb281f)

**Comment by hanjoonwon** [comment]
> Also, you can tell that the model didn't quite learn the shape of the object since the evaluations are really fuzzy (left is the original, right is the predicted one) and that is also confirmed by the depth image directly on top, all is purple so i think the model needs more training ![image](https://private-user-images.githubusercontent.com/90139776/298309084-688ba8d2-7792-4903-8fa8-05c0cffb281f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4MjQwMTMsIm5iZiI6MTcwNTgyMzcxMywicGF0aCI6Ii85MDEzOTc3Ni8yOTgzMDkwODQtNjg4YmE4ZDItNzc5Mi00OTAzLThmYTgtMDVjMGNmZmIyODFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIxVDA3NTUxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliNDQ3ODg0MzRiNjlmODRiNmZmZjIxOGJkNjNmMjc1ODVjZGUzOGI3MzQwMTczNDUzNTQ2N2JmNDY5Mzg0NTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FsVfVgKlaSPLa9KMTbjIOLZa3pNmRtzzRf54GFaEwWw)



Thank you so much. Unlike models like instant-ngp, surface models require more iterations and time.
For the DTU dataset, I got good results starting at 20K, so I only went up to 50K.

**Comment by hanjoonwon** [comment]
> Also, you can tell that the model didn't quite learn the shape of the object since the evaluations are really fuzzy (left is the original, right is the predicted one) and that is also confirmed by the depth image directly on top, all is purple so i think the model needs more training ![image](https://private-user-images.githubusercontent.com/90139776/298309084-688ba8d2-7792-4903-8fa8-05c0cffb281f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNjE5NDYsIm5iZiI6MTcwNjE2MTY0NiwicGF0aCI6Ii85MDEzOTc3Ni8yOTgzMDkwODQtNjg4YmE4ZDItNzc5Mi00OTAzLThmYTgtMDVjMGNmZmIyODFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDA1NDcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmZTdmY2ZmMzdkYWY4NjZjOWQ2OWFkOGM1MGQ0ZDZjODU5ZTMyOTEwMmVmNmUyZDM5NDgyZDczZTlmZjM0YjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YvoNkAPmtHw1bf3dHisyAngu33lcIr-qV336Cu6JzYE)

@davidceka 
https://wandb.ai/ju805604/sdfstudio/runs/hfb0p7g6?workspace=user-ju805604
but got bad result... 
Is this also due to a lack of iterations?

![image](https://github.com/autonomousvision/sdfstudio/assets/125524043/6a255cdd-c9a3-480c-a279-451acddd0cd4)

2.ns-train neus-facto --trainer.load-dir outputs/-workspace-processdata-modifyvidbat/neus-facto/2024-01-22_022432/sdfstudio_models --pipeline.datamanager.train-num-rays-per-batch 1024 --pipeline.model.sdf-field.bias 0.3 --pipeline.model.sdf-field.use-grid-feature False --pipeline.model.mono-depth-loss-mult 0.5 --pipeline.model.mono-normal-loss-mult 0.05 --vis wandb --pipeline.model.sdf-field.inside-outside False --pipeline.model.background-model grid --trainer.steps_per_save 5000 --trainer.steps-per-eval-image 5000 --trainer.max-num-iterations 300000 --experiment-name nomaskmodifyvidbat sdfstudio-data --data /workspace/processdata/modifyvidbat --include_mono_prior True

(Because of the large number of iterations, I proceeded by loading the previous progress)

3.ns-extract-mesh --resolution 512 --bounding-box-min -0.6 -0.6 -0.6 --bounding-box-max 0.6 0.6 0.6 --load-config outputs/nomaskmodifyvidbat/neus-facto/2024-01-22_234123/config.yml --output-path meshoutput/nomask200kbat.ply

here is my data https://drive.google.com/file/d/1WV9wOjWhwsOkBC1g9cNBKaaKMMaYgp5G/view?usp=drive_link
here is my proceesd data
I added a foreground mask and the same thing happened.

**Comment by CuiYan27** [comment]
neus-facto don't need depth and normal. Did you try set the depth and normal loss as 0:
--pipeline.model.mono-depth-loss-mult 0 
--pipeline.model.mono-normal-loss-mult 0
